test/expressions/arrow-expression.js

```diff
@@ -68,5 +98,15 @@ suite('Parser', () => {
     testParseFailure('() + 0', 'Unexpected token "+"');
     testParseFailure('(10) => 0', 'Illegal arrow function parameter list');
     testParseFailure('(10, 20) => 0', 'Illegal arrow function parameter list');
+    testParseFailure('(...a, b) => {}', ErrorMessages.INVALID_LAST_REST_PARAMETER);
+    testParseFailure('(...a, ...b) => {}', ErrorMessages.INVALID_LAST_REST_PARAMETER);
+    testParseFailure('(a, ...b,) => {}', ErrorMessages.INVALID_LAST_REST_PARAMETER);
+    testParseFailure('(async (...a, b) => {})', ErrorMessages.UNEXPECTED_TOKEN(','));
+    testParseFailure('(async (...a, ...b) => {})', ErrorMessages.UNEXPECTED_TOKEN(','));
```

Comment:
If they don't already exist, can you add tests confirming that

```js
async (...a, b);
```
and
```js
async (...a, ...b);
```
parse successfully?